impl/sql/testdb/testdb.go

```diff
@@ -27,12 +26,13 @@ import (
 	ktsql "github.com/google/keytransparency/impl/sql"
 )
 
-var dataSourceURI = flag.String("test_mysql_uri", "root@tcp(127.0.0.1)/", "The MySQL URI to use when running tests")
+var dataSourceURI = "root@tcp(127.0.0.1)/" //  "The MySQL URI to use when running tests"
```

Comment:
Is it really desired to not have any flag control over this value?

Reply:
Options:
1) Have a KT specific flag
2) Export the flag that Trillian defines

Reply:
Going for option 1